packages/odyssey-react-mui/src/Accordion.tsx

```diff
@@ -81,6 +88,8 @@ const Accordion = ({
   return (
     <MuiAccordion
       data-se={testId}
+      data-ods-type="accordion"
+      data-ods-variant={variant}
       defaultExpanded={isDefaultExpanded}
       disabled={isDisabled}
       disableGutters
```

Comment:
@benlister-okta In the borderless version, do we want the gutters? So, if we have 2 of these next to each other, they give each other space?

Reply:
That seemingly only applies when they are open though... wonder if we want to add some margin if these are next to each other. Any thoughts?